rubrik_cdm/data_management.py

```diff
@@ -990,3 +990,109 @@ def sql_live_mount(self, db_name, date, time, sql_instance=None, sql_host=None,
                     mount_name))
 
             return self.post('v1', '/mssql/db/{}/mount'.format(mssql_id), config, timeout)    
+
+    def vsphere_live_unmount(self, mounted_vm_name, force=False, timeout=30):  # pylint: ignore
+        """Delete a vSphere Live Mount from the Rubrik cluster. 
+
+        Arguments:
+            mounted_vm_name {str} -- The name of the Live Mounted vSphere VM to be unmounted. 
+
+        Keyword Arguments:
+            force {bool} -- Force unmount to remove metadata when the datastore of the Live Mount virtual machine was moved off of the Rubrik cluster. (default: {False})
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            dict -- The full response of `DELETE '/vmware/vm/snapshot/mount/{id}?force={bool}'.
+        """
+
+
+        self.log("vsphere_live_unmount: Searching the Rubrik cluster for the Live Mount vSphere VM '{}'.".format(mounted_vm_name))
+        mounted_vm_id = self.object_id(mounted_vm_name, 'vmware', timeout=timeout)
+
+        self.log("vsphere_live_unmount: Getting the vSphere VM mount information from the Rubrik cluster.")
+        mount_summary = self.get('v1', '/vmware/vm/snapshot/mount', timeout=timeout)
+
+        self.log("vsphere_live_unmount: Getting the mount ID of the vSPhere VM '{}'.".format(mounted_vm_name))
+        for mountedvm in mount_summary['data']:
+            if mountedvm['mountedVmId'] == mounted_vm_id:
+                mount_id = mountedvm['id']
+                break
+        else:
+            raise InvalidParameterException("The mounted VM '{}' does not exist, please provide a valid instance".format(mounted_vm_name))
+
+        try:
+            mount_id
+        except NameError:
+            raise InvalidParameterException("The mounted vSphere VM '{}' does exist, please check the name you provided.".format(
+                mounted_vm_name))
+        else:
+            #config = {}
+            #config['force'] = force
+
+            self.log(
+                "vsphere_live_mount: Unmounting the vSphere VM '{}'.".format(mounted_vm_name))
```

Comment:
` vsphere_live_mount: Unmounting the vSphere VM '{}'.".format(mounted_vm_name))`

`vsphere_live_mount` should be `vsphere_live_unmount`

Reply:
Changes made thanks